db/data/20230306180908_downcase_user_emails.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class DowncaseUserEmails < ActiveRecord::Migration[6.1]
+  def up
+    User.all.each do |user|
+      user.update_attribute(:email, user.email.downcase)
+      user.update_attribute(:alternative_email, user.alternative_email&.downcase)
```

Comment:
```suggestion
     user.email = user.email.downcase
     user. alternative_email = user.alternative_email&.downcase
     user.save
```

Reply:
I wanted to skip validations just in case, but if it isn't needed, then 'user.save' will work because it will call callbacks